src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -397,19 +397,19 @@ export class ResultsViewPageStore {
                     this.hotspots ||
                     this.cbioportalCount ||
                     this.cosmicCount ||
-                    this.customBinaryDriverFilter ||
+                    this.isCustomBinaryDriverFilterEnabled ||
                     this.driverTiers.entries().reduce((oneSelected:boolean, nextEntry:[string, boolean])=>{
                         return oneSelected || nextEntry[1];
                     }, false);
 
                 return anySelected;
             },
 
-            get customBinaryDriverFilter() {
-                return this._customBinaryDriverFilter === undefined ? AppConfig.serverConfig.oncoprint_custom_driver_annotation_binary_default : this._customBinaryDriverFilter;
+            get isCustomBinaryDriverFilterEnabled() {
+                return this._isCustomBinaryDriverFilterEnabled === undefined ? AppConfig.serverConfig.oncoprint_custom_driver_annotation_binary_default : this._isCustomBinaryDriverFilterEnabled;
             },
 
-            get customTiersDriverFilter() {
+            get iscustomTiersDriverFilterEnabled() {
```

Comment:
this name still isn't right. this is the *default*, right?

Reply:
First of all, I have fixed the names, I agree with you. Just to clarify why "filter" is here, I think I wrote that name because what the annotations are doing is _filtering_ the mutations in the Oncoprint by type - but of course there is no point on having this name here and it just adds more confusion.

Reply:
Thanks very much, it looks great now. Sorry if I was unpleasant.

Reply:
No problem, the code needs to be clear and understandable for everybody. Thanks a lot for spending time on it 😄